04/index.html

```diff
@@ -4,27 +4,162 @@
+                  <li class="nested__item"><a href="#" class="dropdown__link">Product positioning</a></li>
+                </ul>
+              </li>
+            </ul>
+          </li>
+          <li class="menu__item">
+            <a href="#" class="menu__link">Contact</a>
+            <ul class="menu__dropdown dropdown">
+              <li class="dropdown__item">
+                <span><a href="#" class="dropdown__link">E-mail</a></span>
+                <img
+                  src="./images/icon-arrow-down.svg"
+                  alt="Arrow right"
+                  class="dropdown__arrow"
+                />
+                <ul class="dropdown__nested">
+                  <li class="nested__item"><a href="#" class="dropdown__link">Send an e-mail</a></li>
                 </ul>
-            </li>
-            <li>
-                <a href="#">kontakt</a>
-            </li>
+              </li>
+              <li class="dropdown__item">
+                <span><a href="#" class="dropdown__link">Social media</a></span>
+                <img
+                  src="./images/icon-arrow-down.svg"
+                  alt="Arrow right"
+                  class="dropdown__arrow"
+                />
+                <ul class="dropdown__nested">
+                  <li class="nested__item"><a href="#" class="dropdown__link">Facebook</a></li>
+                  <li class="nested__item"><a href="#" class="dropdown__link">Twitter</a></li>
+                </ul>
+              </li>
+          </li>
         </ul>
-    </nav>
+      </nav>
+
+      <!-- navigation mobile -->
+
+  <a href="#overlay" class="open-mobile-menu" id="close"><img src="./images/icon-hamburger.svg" alt="Open mobile menu - hamburger" class="open-mobile-menu__img"></a>
+
+  <div class="overlay" id="overlay">
+    <div class="mobile">
+      <a href="#close" class="overlay__close">&times;</a>
+      <ul class="mobile__menu">
+          <li class="mobile-menu__item"><a href="" class="mobile-menu__link">Start <img
+              src="./images/icon-arrow-down.svg"
+              alt="Arrow right"
+              class="mobile__arrow"
+            /> </a>
+          </li>
+          <hr class="mobile-hr">
+          <li class="mobile-menu__item">
+              <a href="#" class="mobile-menu__link">Offer <img
+                  src="./images/icon-arrow-down.svg"
+                  alt="Arrow right"
+                  class="mobile__arrow"
+                  /> </a> 
+                  <ul class="mobile-dropdown">
+                    <li class="mobile-dropdown__item">
+                      <span><a href="#" class="mobile-dropdown__link">Websites</a></span>
+                      <img
+                        src="./images/icon-arrow-down.svg"
+                        alt="Arrow right"
+                        class="mobile__arrow-l1"
+                      /> 
+                      <ul class="mobile-dropdown-l2">
```

Comment:
W przypadku wersji mobilnej nie jesteśmy w stanie się dostać to elementu zagnieżdżonego.
Pamiętaj, że tutaj `:hover` jest tutaj interpretowane jako kliknięcie (testuj na wersji mobilnej devtools).